source/includes/_endpoint-progress.md

```diff
@@ -16,7 +11,7 @@ GET /progress/{id}/ HTTP/1.1
 ```json
 {
     "element": "shoji:view",
-    "self": "https://api/progress/{id}/",
+    "self": "https:/beta.crunch.io/api/progress/{id}/",
     "value": {
         "progress": "22.5",
         "message": "exported 2 variables",
```

Comment:
Is `dataset_id` required? Is that `{id}` the same as the `{id}` in the URL template in `self`? (I suspect the answer is "no" to both.)